godtools/XMLContent/KGP/kgp-5.xml

```diff
@@ -100,7 +99,7 @@
                 </content:text>
             </paragraph>
             <paragraph>
-                <content:button type="event" value="kgp-6">
+                <content:button type="event" value="kgp-6,send-information-modal">
```

Comment:
Once Goers gets the schema validation PR completed and merged I'll work on updating the XSD for this.